server/apps/packages/takes_package_service.py

```diff
@@ -199,8 +204,7 @@ def get_first_take_in_takes_package(self, item):
         """
         package = self.get_take_package(item)
         if package:
-            groups = package.get('groups', [])
-            refs = next((group.get('refs') for group in groups if group['id'] == 'main'), None)
+            refs = self.__get_package_refs(package)
```

Comment:
That's it true. Nothing is private.

I considered single underscore but looking at the implementation I would never subclass it hence I used for double underscore. 

Sure will change to single underscore.